decentraland/unity-renderer
# Feat: merged channels and notifications

## What does this PR change?

Merges channels and notifications 

...

## How to test the changes?

1. Go to: https://play.decentraland.zone/?NETWORK=mainnet&realm=dg&position=101%2C100&ws=ws%3A%2F%2Flocalhost%3A5000%2Fdcl&ENABLE_NOTIFICATION_CHAT=&island=I1qv21
2. Verify that channels and notifications work as expected

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
